File: django-master/collection/dictionarypgms/wordcount.py
text="hello,hai,hello,hai,how,how,are,hello"
words=text.split(",")
print(words)
dict={}
for word in words:
    if word in dict:
        dict[word]+=1
    else:
        dict[word]=1
print(dict)

#to find highest occuring word
data=sorted(dict,key=dict.get,reverse=True)
print(data)
maxword=max(dict,key=dict.get)
print(maxword)


#to find least occuring word
minword=min(dict,key=dict.get)
print(minword)

File: django-master/html section/javascript/objects/wordcount.js
var text="hello hello hai how are";
//word count

var words=text.split(" ")
//[hello hello hai how are]====> now its an array of separate words

var obj={}
for(let word of words){
    if(word in obj){
        obj[word]+=1
    }
    else{
        obj[word]=1
    }
}
console.log(obj)